Sodium/PWHash.swift

```diff
@@ -15,18 +15,18 @@ public struct PWHash {
 
 extension PWHash {
     public enum Alg {
-        case Default
-        case Argon2I13
-        case Argon2ID13
+        case `default`
```

Comment:
Avoid ticks and name it `defaultAlgorithm` as @jedisct1 [suggested](https://github.com/jedisct1/swift-sodium/pull/184#issuecomment-489597858).